tests/bluetooth/bsim_bt/bsim_test_audio/test_scripts/mcs_mcc.sh

```diff
@@ -32,8 +32,9 @@ Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_audio_prj_conf \
 Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_audio_prj_conf \
   -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=1 -testid=mcs -rs=23
 
+# Simulation time should be larger than the WAIT_TIME in common.h
 Execute ./bs_2G4_phy_v1 -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} \
-  -D=2 -sim_length=20e6 $@
+  -D=2 -sim_length=31e6 $@
```

Comment:
31 seconds seems oddly specific. Could 30s be used? 

We can also just set it to 60s; it's not like a successful test will be affected by these

Reply:
The WAIT_TIME is 30 seconds, so I wanted this one to be slightly longer to be sure that WAIT_TIME hits first.
I can set it to 60, but there is not really much point, WAIT_TIME will stop it after 30 anyway.

Reply:
Tried setting both WAIT_TIME and the simulation length to the same value, and the WAIT_TIME actually triggers first.
But I can not see that we have any guarantee that this will (always) happen, so set all the simulation lengths to 60 s.